react/features/toolbox/components/web/Toolbox.js

```diff
@@ -278,6 +280,11 @@ class Toolbox extends Component<Props, State> {
                 character: 'S',
                 exec: this._onShortcutToggleFullScreen,
                 helpDescription: 'keyboardShortcuts.fullScreen'
+            },
+			this._shouldShowButton('videoquality') && {
```

Comment:
This like there's also de-registration (APP.keyboardshortcut.unregisterShortcut) that should occur.